packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -188,7 +189,7 @@ describe('Marketplace page - layout', () => {
     await waitForReload();
 
     const providersTab = screen.getByRole('tab', { name: /providers/i });
-    userEvent.click(providersTab);
+    fireEvent.click(providersTab);
```

Comment:
you can't use `userEvent`?

Reply:
You can, but it was easier to use `fireEvent`:

1. it is sufficient in this case I think
2. it doesn't require the test to be async

If you want I can revert back ...

Reply:
Nah that's fine just was curious why. It does seem a bit overkill to make the test async for the one click. Up to you